compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/ComposeScenePointer.skiko.kt

```diff
@@ -127,7 +127,8 @@ internal fun PointerInputEvent(
             it.type,
             issuesEnterExit = it.type == PointerType.Mouse,
             historical = it.historical,
-            scrollDelta = scrollDelta
+            scrollDelta = scrollDelta,
+            it.position
```

Comment:
Let's use parameter name for this line too

Reply:
Done